core/src/main/java/feast/core/model/FeatureGroupInfo.java

```diff
@@ -111,4 +111,23 @@ public FeatureGroupDetail getFeatureGroupDetail() {
             .setLastUpdated(TypeConversion.convertTimestamp(this.getLastUpdated()))
             .build();
   }
+
+  /**
+   * Checks if this is eq to the other given feature group
+   *
+   * @param otherFeatureGroup
+   * @return boolean
+   */
+  public boolean eq(FeatureGroupInfo otherFeatureGroup) {
```

Comment:
Isn't it possible (and perhaps cleaner) to use the built in .equals() method that all Java objects have? 

https://docs.oracle.com/javase/7/docs/api/java/lang/Object.html#equals(java.lang.Object)